frontend/src/components/headerSection/header.css

```diff
@@ -85,7 +82,6 @@
 @media (max-width: 420px /*naisu*/) {
     .app-header {
         min-height: 50vh;
-        background-size: 100%;
```

Comment:
This was intentional to make sure mobile views don't see a half cut off background.

https://github.com/Manotomo-Alliance-Support-Squad/WWS/issues/26#issuecomment-848002005

Reply:
Updated